react-app/package.json

```
@@ -1,12 +1,14 @@
 {
-  "name": "too_many_choco",
+  "name": "react-app",
```

Please don't change the React app name : 
```suggestion
  "name": "too_many_choco",
```